packages/babel-plugin-minify-mangle-names/src/index.js

```diff
@@ -311,6 +308,19 @@ class Charset {
   }
 }
 
+// convert value to Map object
+function toMap(value) {
```

Comment:
Since JS has a [Map](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Map) let's call this `toObject`